onboarder/installation/define_environment_variables/dot_env_component.rst

```diff
@@ -7,3 +7,13 @@ into the PIM as it's a standard Symfony framework component. It is also used by
 
 Edit the ``.env`` file at your project root directory and add the environment variables values provided on your
 Onboarder project page on the Partner Portal.
+
+.. code-block:: bash
```

Comment:
Why do you add those variables to the Onboarder documentation? They are related to the PIM.

Reply:
@momoss Do you know why we have to tell the customer that they have to define those variables? If they are not defined, the PIM does not work at all